ICAFIX/hcp_fix_multi_run

```diff
@@ -377,7 +424,7 @@ log_Msg "PWD: $DIR"
 # to the matlab path (i.e., the ciftiopen.m, ciftisave.m functions).
 # The pipelines CIFTI I/O functions have been moved to a subdirectory, and the SetUp... script selects what to use with HCPCIFTIRWDIR.
 # AND since 'addpath' *prepends* to the matlab path, the version in HCPCIFTIRWDIR will therefore take precedence.
-ML_PATHS="addpath('${HCPPIPEDIR}/global/fsl/etc/matlab'); addpath('${HCPCIFTIRWDIR}'); addpath('${HCPPIPEDIR}/global/matlab/icaDim'); addpath('${HCPPIPEDIR}/global/matlab'); addpath('${this_script_dir}/scripts');"
+ML_PATHS="addpath('${HCPPIPEDIR}/global/fsl/etc/matlab'); addpath('${HCPCIFTIRWDIR}'); addpath('${HCPPIPEDIR}/global/matlab/icaDim'); addpath('${HCPPIPEDIR}/global/matlab/icasso122'); addpath('${HCPPIPEDIR}/global/matlab'); addpath('${this_script_dir}/scripts');"
```

Comment:
With the removal of the use of `call_matlab.sh` to invoke `functionhighpassandvariancenormalize` we probably should remove mention of `call_matlab` from this script.  [Mentioned in the comments a couple lines earlier to here (L421 currently) and later (L730 currently) as well].

Reply:
Sounds reasonable.